### PR TITLE
Bug report template

### PR DESCRIPTION
**Pull request by colmtuite**
Update the bug report template to something more inline with industry standard, providing a single textarea for much easier editing with less cognitive overload.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1.bug.yml [hunk 1]
```diff
@@ -1,65 +1,38 @@
...
+
+Mac OS/Windows/Other etc.
+
+### Which assistive tech are you using (if applicable)?
+
+Voiceover/JAWS etc.
+
+### Current Behavior
+
+Provide a clear and concise description of the current behaviour you're experiencing. If applicable, provide screenshots and/or videos.
```
Comment (michaldudak) [inline .github/ISSUE_TEMPLATE/1.bug.yml:26]: behavior / behaviour

File: .github/ISSUE_TEMPLATE/1.bug.yml [hunk 2]
```diff
@@ -1,65 +1,38 @@
...
+
+### Expected behavior
+
+Provide a clear and concise description of the expected behaviour.
```
Comment (michaldudak) [inline .github/ISSUE_TEMPLATE/1.bug.yml:30]: Same

File: .github/ISSUE_TEMPLATE/1.bug.yml [hunk 3]
```diff
@@ -1,65 +1,38 @@
...
+Chrome/Safari/Firefox/Safari iOS etc.
+
+### Which OS are you using?
+
+Mac OS/Windows/Other etc.
+
+### Which assistive tech are you using (if applicable)?
+
+Voiceover/JAWS etc.
+
```
Comment (vladmoroz) [inline .github/ISSUE_TEMPLATE/1.bug.yml:23]: I think this stuff has to go last

Most issues won't be browser or OS specific and whoever comes to this page is going to be interested in the issue description first and env specific last

File: .github/ISSUE_TEMPLATE/1.bug.yml [hunk 4]
```diff
@@ -1,65 +1,38 @@
...
+### Current Behavior
```
Comment (vladmoroz) [inline .github/ISSUE_TEMPLATE/1.bug.yml:24]: Turbo nit: consistent subheading casing

File: .github/ISSUE_TEMPLATE/1.bug.yml (hunk 1 above)
Comment (colmtuite) [reply .github/ISSUE_TEMPLATE/1.bug.yml:26]: It's US spelling

File: .github/ISSUE_TEMPLATE/1.bug.yml (hunk 2 above)
Comment (colmtuite) [reply .github/ISSUE_TEMPLATE/1.bug.yml:30]: It's US spelling

File: .github/ISSUE_TEMPLATE/1.bug.yml (hunk 1 above)
Comment (michaldudak) [reply .github/ISSUE_TEMPLATE/1.bug.yml:26]: I mean it's "behavior" in the title and "behaviour" in the paragraph. Let's settle on one.

**Review by vladmoroz — APPROVED** [review]

**Pull request merged**